install/ansible-automation-controller/install.yml

```diff
@@ -0,0 +1,15 @@
+id: ansible-automation-controller
+name: Ansible Automation Controller
+title: Ansible Automation Controller monitoring
+description: |
+  The Ansible Automation Controller integration will enable you to monitor and analyze your Ansible Automation Controller infrastructure.
```

Comment:
The Red Hat Ansible Automation Controller integration will enable you to monitor and analyze your Ansible Automation Controller infrastructure.

Reply:
@raahulbasu ,

Reply:
Wouldn't it be more correct to write something like:

"New Relic's Ansible Automation Controller integration enables you to monitor and analyze your Ansible infrastructure."

or even

"Use New Relic's infrastructure agent to monitor and analyze your Ansible Automation Controller performance."?